ui-tests/src/test/resources/features/integrations/gmail.feature

```diff
@@ -0,0 +1,112 @@
+@gmail
+Feature: Google mail Connector
+
+  Background: Clean application state
+    Given clean application state
+    Given reset content of "contact" table
+    Given delete emails from "jbossqa.fuse@gmail.com" with subject "syndesis-test"
+    Given log into the Syndesis
+    Given created connections
+      | Google Mail | QE Google Mail | My GMail Connector | SyndesisQE Slack test |
+    And navigate to the "Home" page
+
+#
+#  1. Send an e-mail
+#
+  @gmail-send
+  Scenario: Send an e-mail
+
+    # create integration
+    And click on the "Create Integration" button to create a new integration.
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+    When select the "PostgresDB" connection
+    And select "Periodic SQL Invocation" integration action
+    Then check "Done" button is "Disabled"
```

Comment:
this one probably shouldn't be here, as you are not testing creating db connection